wiki/Tournaments/GO/3/en.md

```diff
@@ -141,31 +140,61 @@ The Game\_Osu Tourney 3 is run by various community members.
+  - [BUTAOTOME - Mesen (BarkingMadDog) \[EXPERT\]](https://osu.ppy.sh/beatmapsets/865750#osu/1810154)
+  - [Uchu Conbini - EverythingChanges (Halgoh) \[Phase\]](https://osu.ppy.sh/beatmapsets/959415#osu/2008658)
+  - [Nanjo Yoshino x Kayano Ai (as Shirabe Tsukuyomi x Kirika Akatsuki) - Edge Works of Goddess ZABABA (Anxient) \[LiNKER\]](https://osu.ppy.sh/beatmapsets/600902#osu/1269601)
+  - [Otokaze - Saigetsu (Short Ver.) (Yusomi) \[Years\]](https://osu.ppy.sh/beatmapsets/746240#osu/1572866)
+  - [xi remixed by cosMo@bousouP - FREEDOM DiVE [METAL DIMENSIONS] (Cherry Blossom) \[METAL DIMENSIONS\]](https://osu.ppy.sh/beatmapsets/905158#osu/1889232)
+  - [MAX MAXIMIZER vs. DJ TOTTO - Rebellio (Realazy) \[OUTRAGE\]](https://osu.ppy.sh/beatmapsets/744636#osu/1846898)
+  - [FujuniseikouyuuP - Sayonara Lechenaultia (Meg, Shirasaka Koume) \[Collab EXTRA\]](https://osu.ppy.sh/beatmapsets/703769#osu/1488898)
+  - [banshi - -Yashiro- (Yusomi) \[Hanabi Dance\]](https://osu.ppy.sh/beatmapsets/573571#osu/1215044)
+- Hidden
+  - [ETIA. - Nihonshiki Koukaku-OukaRanman- (den0saur) \[Black Another\]](https://osu.ppy.sh/beatmapsets/604664#osu/1277591)
+  - [Camellia - Upload Your Mind :: Download My Soul (Slaked) \[Lead Me :: Follow Me -Take Me to the Stars-\]](https://osu.ppy.sh/beatmapsets/914449#osu/1909963)
+  - [Sakuzyo - Stoicism (dsco) \[Jazz!\]](https://osu.ppy.sh/beatmapsets/705318#osu/1491800)
+  - [Hatsuki Yura - Shoujo to Ougonryuu no Monogatari (Garden) \[Salvation\]](https://osu.ppy.sh/beatmapsets/719158#osu/1518701)
+- HardRock
+  - [siromaru + cranky - conflict (Shirasaka Koume) \[,\]](https://osu.ppy.sh/beatmapsets/576015#osu/1219580)
+  - [Mio Yamazaki - Noise (English ver.) (Akitoshi) \[Extreme\]](https://osu.ppy.sh/beatmapsets/702892#osu/1487213)
+  - [USAO - BroGamer (Avena) \[Avena's Ultra\]](https://osu.ppy.sh/beatmapsets/635636#osu/1395749)
+  - [Eve - Gekka\*Midare Botan (Meg, Roll\_Pan) \[Meg&Roll\_Pan's E-Xtra\]](https://osu.ppy.sh/beatmapsets/530685#osu/1124857)
+- DoubleTime
+  - [kanonxkanon - Calendula Requiem (smallboat) \[Shiki Revive\]](https://osu.ppy.sh/beatmapsets/172293#osu/416492)
+  - [Tsukasa - Heaven's Race Guitar Style (La Cataline, Cherry Blossom) \[Collab\]](https://osu.ppy.sh/beatmapsets/41974#osu/132260)
+  - [FELT - Puppet in the Dark (Mackenzie) \[Out Of Place ARTifactS\]](https://osu.ppy.sh/beatmapsets/326636#osu/725198)
+  - [The Pretty Reckless - Kill Me (Zero\_\_wind) \[Kill!!!\]](https://osu.ppy.sh/beatmapsets/97187#osu/260006)
+- FreeMod
+  - [t+pazolite - Magical Higan Tour 2009 (DJNightmare, Nakagawa-Kanon) \[Collab\]](https://osu.ppy.sh/beatmapsets/40609/#osu/128775)
+  - [5 Step Sound Team - Never Say Die (P o M u T a) \[Die\]](https://osu.ppy.sh/beatmapsets/292599#osu/658488)
+  - [join. SELEN - DADADADADADADA (Sharu) \[Sharu's Extreme\]](https://osu.ppy.sh/beatmapsets/773755#osu/1632692)
+  - [Luschka - Kami no Kotoba (Jounzan) \[Jounzan's Special\]](https://osu.ppy.sh/beatmapsets/817667#osu/1714634)
```

Comment:
still lost as to why the name in parenthesis is anything but the mapset host, all this format does it make it harder to search for the map (picked a random example but plenty of the maps do this)

this format is supposed to go Artist - Title (Creator) [Version], it's misleading to put someone who's not the mapset creator into that field. a computer should be able to make this text